src/main/java/seedu/address/logic/commands/DeleteTransactionCommand.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Transaction;
+
+/**
+ * Deletes a transaction from a person's transaction list using its specified index.
+ */
+public class DeleteTransactionCommand extends Command {
+    public static final String COMMAND_WORD = "deletet";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the transaction identified by the index number used in the displayed transaction list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_TRANSACTION_SUCCESS = "Deleted transaction %1$s\\nfrom %2$s";
```

Comment:
I think there's an extra `\` here so the message format is a little off
![image](https://github.com/user-attachments/assets/db285169-7d06-4f17-a4ef-88a47fccdd52)